CMSgov/onemac
# OY2 21899: Display Role based Status on package dashboard and details pages

Story: https://qmacbis.atlassian.net/browse/OY2-21899
Endpoint: https://d2b7wxo785vsd2.cloudfront.net

### Details
State users and CMS users see different package statuses

### Changes
- Migrated the sample data from product-change-request-2 into the feature branch change-request table
- adjusted the json files to point to change-request table (the default)
- Updated the Withdraw Action to update status to "Withdrawal Requested"
- Added flag on the role config for "isCMSUser" to designate which status mapping to use
- added lib/status-lib to hold the status mapping details
- Updated getDetail to convert the currentStatus to the status for the role
- getMyPackages now converts the currentStatus to the status for the role
- adjusted RAI Response submissions to update status to "Submitted"
- made the ONE_MAC statuses more "generic" and added in the new CMS statuses to be displayed
- removed the initial status list from Package List

### Test Plan
1. Log in as a state submitter
2. Verify the status labels in the package dashboard are the state user values
3. Verify the status labels in the package details are the state user values
4. Verify that Submitting an RAI returns the package to Submitted status
5. Verify that Withdrawing a package puts the package into Withdrawal Requested status
6. Log out and log in as a cms user (or, I used a different browser so I could view the same package with the two roles)
7. Verify the status labels on the package dashboard are the cms user values
8. Verify the status labels in the package details are the cms user values
